src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -21,6 +21,100 @@
+                      <dd>{{ t('common.formats.integer', {value: 1000}) }}</dd>
+                    </div>
+                  </dl>
+                </template>
+                <ServiceTrafficGroup
+                  type="inbound"
+                >
+                  <ServiceTrafficCard
+                    v-for="item in data.inbounds"
+                    :key="`${item.name}`"
+                    :protocol="item.protocol"
+                    :total="item.metrics.external?.upstream_rq_2xx || 0"
+                    :success="item.metrics.external?.upstream_rq_completed || 0"
+                  >
+                    {{ item.name }}
+                  </ServiceTrafficCard>
+                </ServiceTrafficGroup>
+              </DataPlaneTraffic>
+              <DataPlaneTraffic
+                v-if="typeof data !== 'undefined'"
+              >
+                <template #title>
+                  <GatewayIcon
+                    display="inline-block"
+                    decorative
+                    :size="KUI_ICON_SIZE_30"
+                  />
+                  <span>Outbounds</span>
+                </template>
+                <template #data>
+                  <dl>
+                    <div>
+                      <dt class="passthrough">
+                        {{ t('services.components.service_traffic.passthrough', {}, {defaultMessage: 'Passthrough Requests'}) }}
+                      </dt>
+                      <dd>{{ t('common.formats.integer', {value: 1000}) }}</dd>
+                    </div>
+                    <div>
+                      <dt class="outbounds">
+                        {{ t('services.components.service_traffic.mesh', {}, {defaultMessage: 'Mesh Requests'}) }}
+                      </dt>
+                      <dd>{{ t('common.formats.integer', {value: 1000}) }}</dd>
+                    </div>
+                  </dl>
+                </template>
+                <ServiceTrafficGroup
+                  type="passthrough"
+                >
+                  <ServiceTrafficCard
+                    :protocol="`unknown`"
+                    :total="data.passthrough.reduce((prev, item) => prev + (item.metrics.downstream_cx_total || 0), 0)"
+                    :success="982"
+                  >
+                    Non mesh traffic
+                  </ServiceTrafficCard>
+                </ServiceTrafficGroup>
+                <ServiceTrafficGroup
+                  type="outbound"
+                >
+                  <ServiceTrafficCard
+                    v-for="item in data.outbounds"
+                    :key="`${item.name}`"
+                    :protocol="item.protocol"
+                    :total="item.protocol === 'http' ? item.metrics.downstream_rq_http1_total : item.metrics.downstream_cx_total"
+                    :success="item.protocol === 'http' ? item.metrics.downstream_rq_2xx : item.metrics.downstream_cx_success || -1"
+                  >
```

Comment:
I'm kinda guessing on which properties to use here (or potentially sums/aggregations of properties?), it also depends on protocol from what I see and remember from when we were chatting through this.